### PR TITLE
[R1497] - native 3ds integration

### PR DESCRIPTION
**Pull request by ryft-rkirby**
Implements the required changed based on the latest API to facilitate 3ds v2 via the native SDK rather than a web view.

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -25,6 +25,18 @@ let package = Package(
     ],
     dependencies: [],
     targets: [
+        .binaryTarget(
```
Comment (ryft-rkirby) [inline Package.swift:28]: I've bundled these within the repo under `/frameworks` due to the 3ds sdk required authentication to access.
Note that the first 2 here and dependencies for the 3ds sdk

File: RyftCore/Source/Api/AttemptPaymentRequest.swift [hunk 2]
```diff
@@ -50,7 +52,28 @@ public struct AttemptPaymentRequest {
...
+            clientSecret: clientSecret,
+            cardDetails: nil,
+            walletDetails: nil,
+            billingAddress: nil,
+            customerDetails: nil,
+            threeDsRequestDetails: PaymentRequestThreeDsDetails.defaultValue,
+            paymentMethodOptions: nil,
+            paymentMethod: PaymentRequestPaymentMethod(
+                id: paymentMethodId,
+                cvv: cvv
```
Comment (mike-ridge) [inline RyftCore/Source/Api/AttemptPaymentRequest.swift]: this should be `cvc` based on the api spec: https://api-reference.ryftpay.com/#tag/Payments/operation/paymentSessionAttemptPayment

File: RyftUI/Source/Controllers/RyftDropInPaymentViewController.swift [hunk 3]
```diff
@@ -198,15 +204,21 @@ public final class RyftDropInPaymentViewController: UIViewController {
     }
 
     public func handleRequiredAction(
-        returnUrl: String,
+        returnUrl: URL?,
```
Comment (mike-ridge) [inline RyftUI/Source/Controllers/RyftDropInPaymentViewController.swift:207]: might be worth updating the PaymentSession model `returnUrl` to a URL type too, or update the example here in the README to transform the string to a URL: https://github.com/RyftPay/ryft-ios#implementing-the-ryftdropinpaymentdelegate

Comment (ryft-rkirby) [reply RyftUI/Source/Controllers/RyftDropInPaymentViewController.swift:207]: yeah good spot, I'll have a look at decoding the JSON on the `PaymentSession` so it's already a `URL` in the next change

**Review by mike-ridge — APPROVED** [review]

**Pull request merged**